src/app/query/bitcoin/balance/btc-taproot-balance.hooks.ts

```diff
@@ -18,7 +20,7 @@ export function useCurrentTaprootAccountUninscribedUtxos() {
     const inscriptions = query.data?.pages?.flatMap(page => page.inscriptions) ?? [];
     return filterUtxosWithInscriptions(
       inscriptions,
-      utxos.filter(utxo => utxo.status.confirmed)
+      utxos.filter(utxo => utxo.status.confirmed && utxo.value > RETRIEVE_UTXO_DUST_AMOUNT)
```

Comment:
Personal pref, but more readable as separate filters imo

```suggestion
      utxos
        .filter(utxo => utxo.status.confirmed)
        .filter(utxo => utxo.value > RETRIEVE_UTXO_DUST_AMOUNT)
```

Reply:
put to sep variable also

Reply:
and increased dust amount. should be 10000